### ISSUE TITLE
Resonating Wand does NOT detect/show sparks for Rock Crystal deposits

### ISSUE THREAD

**Issue by Sorbe** [Darkosto/SevTech-Ages#1811]
I walked around for over 10 hours of game time AT NIGHT in the overworld looking for the documented and supposed sparks from underground Rock Crystal veins.   There apparently were no Rock Crystals in the overworld because the world's chunks were created with an early version of the game.

I was very surprised when Rock Crystal veins surface marker were trivially found in the hunting dimension and even more surprising when I found that the Resonating Wand didn nothing at all to reveal them.  Not on the surface, and not even right next to the ore at bedrock.

This was a giant user confusion issue because the in-game book and item are completely misleading.

**Comment by JRiebling** [comment]
That functionality is confirmed to be disabled in this modpack. You should be able to find Rock Crystals semi-easily by rock samples on the surface and digging down (though it won't necessarily be right below the samples, from my experience).

**Comment by savosdarkran** [comment]
So do rock crystals actually spawn in the overworld?  I started with 3.0.4 ( should have fixed the issue where rock crystal was not being generated ).

**Comment by Skullywag** [comment]
Might just be my game but having a hunting dimension set to plains had A LOT rock crystal deposits.

**Comment by Sorbe** [comment]
Found the rock crystals, but I had to go out about a 1000 units in the overworld so that I got fresh chunks.  My overworld was created during beta and had only the very basic ores visible in close to the world spawn.   Once I went out far enough away on 3.0.6 I got both Rock Crystals and Lapis, both of which were totally absent.

As for my original bug -- the wand is completely useless for detecting Rock Crystals in Sevtech, you MUST wander about and find the surface samples.  If this is "By Design', I agree that the bug can be closed.  Seems like a confusing and wholly unnecessary NERF imo.
